docs/cmd-deps-convert.md

```diff
@@ -1,8 +1,15 @@
 # dephell deps convert
 
-Convert dependencies between formats. Dephell will automagically lock them if needed and resolve all conflicts. Dependency resolution is the strongest side of the DepHell, pip and pipenv has no any dependency resolution that would be smart enough.
+Convert dependencies between formats. Dephell will automagically lock them if needed and resolve all conflicts.
 
-You can specify input and output in three different ways:
+Dephell uses four pieces of information for conversion:
+1. `--from-format`: The format to convert from (e.g. `poetry`)
+1. `--from-path`: The path to the file to read from (e.g. `pyproject.toml`)
+1. `--to-format`: The format to convert to (e.g. `setuppy`)
+1. `--to-path`: The path to the file where the result should be put (e.g. `setup.py`)
+    1. You can provide the special case 'stdout' to this option to output to the screen instead of a file.
```

Comment:
I think it should be just sentenced, not lonely indented subpoint in the list :)

Reply:
Agreed, I'll make it less lonely.